data/json/recipes/recipe_armor.json

```diff
@@ -7815,6 +7842,94 @@
     ]
 },{
   "type" : "recipe",
+  "result": "armor_lightplate",
```

Comment:
This whole recipe is pretty redundant. It would make sense if it combined all sorts of greaves, gauntlets etc. into one armor, but just making entire armor around cuirass won't see much use.